src/js/common/schemaform/Address.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import _ from 'lodash';
+import { set } from 'lodash/fp';
+
+import ErrorableSelect from '../components/form-elements/ErrorableSelect';
+import ErrorableTextInput from '../components/form-elements/ErrorableTextInput';
+import { countries, states } from '../utils/options-for-select';
+
+/**
+ * Input component for an address.
+ *
+ * No validation is provided through a currently stubbed isAddressValid function.
+ */
+class Address extends React.Component {
+  constructor(props) {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+    this.handleBlur = this.handleBlur.bind(this);
+    this.state = { value: props.formData, touched: { street: false, city: false, state: false, country: false, postalCode: false } };
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId('address-input-');
+  }
+
+  handleBlur(field) {
```

Comment:
This will probably need to change some. I think it can be one line:

`this.props.onBlur([field])`